module/app/app.go

```diff
@@ -410,6 +410,7 @@ func NewGravityApp(
 		app.bankKeeper,
 		app.slashingKeeper,
 		sdk.DefaultPowerReduction,
+		app.ModuleAccountNames(),
```

Comment:
I'm thinking it might be safe to pass in a explicit list of accounts that the Gravity Module is allowed to send to instead of all module accounts.

See this part of the BankKeeper spec https://github.com/cosmos/cosmos-sdk/blob/58597139fa0fb9e9be60deebee3df1663aa2cfaf/x/bank/spec/README.md#module-accounts

Reply:
Fair point, I could pick an example module here in the "sample" app that gravity uses. Perhaps renaming the field "AllowedModuleAccounts" or something like that? The handling of the map doesn't need to change.

Reply:
I agree with the proposed approach